redbeams-api/src/main/java/com/sequenceiq/redbeams/doc/ModelDescriptions.java

```diff
@@ -2,58 +2,79 @@
 
 public final class ModelDescriptions {
 
-    public static final String ID = "ID of the resource";
-    public static final String CRN = "CRN of the resource";
-    public static final String DESCRIPTION = "Description of the resource";
-    public static final String CREATION_DATE = "Creation date / time of the resource, in epoch milliseconds";
-    public static final String ENVIRONMENT_CRN = "Crn of the environment of the resource";
+    public static final String ALLOCATE_DATABASE_SERVER_REQUEST = "Request to allocate a new database server in a provider";
```

Comment:
NIT: This one says "Request to..." and the next one says, "Request for..." Please make consistent.

Reply:
Will change in the next PR!

Reply:
Actually, since a rebase is necessary, I fixed this now.